server/queue/queue-rabbitmq/src/test/resources/json/mailqueueview/configuration/configuration_changed.json

```diff
@@ -4,5 +4,6 @@
   "aggregateKey": "aggKey",
   "bucketCount": 1,
   "updateBrowseStartPace": 1000,
-  "sliceWindow": 3600.000000000
+  "sliceWindow": 3600.000000000,
+  "sizeMetricsEnabled": false
```

Comment:
Touching the format of a persisted events should raise the following questions:
 - How about past events reading with the old format?
 - And... Is it really necessary?